README.md

```diff
@@ -17,6 +19,10 @@ The MSAL preview library for iOS and macOS gives your app the ability to begin u
 * How to call the Microsoft Graph API
 * How to sign a user out of your application
 
+## Scenario
+
+This app is a multi-tenant app meaning it can be used by any Azure AD tenant or Microsoft Account.  It demonstrates how a developer can build apps to connect with enterprise users and access their Azure + O365 data via the Microsoft Graph.  During the auth flow, end users will be required to sign in and consent to the permissions of the application, and in some cases may require an admin to consent to the app.  The majority of the logic in this sample shows how to auth an end user and make a basic call to the Microsoft Graph.
+
 ## Example
 
 ```swift
```

Comment:
Don't you recommend the patter acquireTokenSilent / acquireToken ?

Reply:
I'm trying to limit the code displayed. The pattern is discussed in the code itself. I can update if we want to show the pattern.

Reply:
I think it's better if we tell the message we want people to remember?